app/views/users/_user_list.html.erb

```diff
@@ -14,7 +14,7 @@
                       <p><%= presenter.title_list %></p>
                   <% end %>
                   <% if user.country_name %>
-                      <p><%= presenter.country_name %></p>
+                      <p title="<%= presenter.country_name unless presenter.country_name.length < 15 %>"><%= truncate(presenter.country_name, length:14) %></p>
```

Comment:
ah, follow up on this as I look closer - you are setting tooltips only when they are long - why not reduce the complexity and set them all the time, then we don't have to worry about testing for both cases - just have them always there?